libmikmod/PSPBUILD

```diff
@@ -9,34 +9,22 @@ license=('GPL2.1')
 depends=()
 makedepends=()
 optdepends=()
```

Comment:
Please increase the pkgrel if the package version isn't changed

Reply:
Done